assets/scripts/datalayer/common.js

```diff
@@ -103,6 +103,14 @@ document.shopAnalytics = {
       }
       window.dataLayer.push(event_data);
     }
+  },
+
+  event: {
+    click: {
+      login: '.woocommerce-form-login button[name="login"]',
+      register: '.woocommerce-form-register button[name="register"]',
+      registerOnCheckoutSubmit: '.woocommerce-checkout #place_order'
```

Comment:
1. There is no "submit" suffix in the other keys, so we should remove it here for consistency.

2. The third item registers an event for the generic "order now" button in the checkout, regardless of whether a user registers or not.  I guess we could use one of the body classes to only apply the behavior to anonymous users?